app/templates/view_supplier_users.html

```diff
@@ -5,129 +5,164 @@
 {% extends "_base_page.html" %}
 
 {% block page_title %}
-  {{ supplier.name }} – Digital Marketplace admin
```

Comment:
pycharm hates the templates/html. Often trips me up. Should be a little better now.